interface-spec.md

```diff
@@ -4,10 +4,10 @@
 **Purpose:** This document proposes a uniform interface to represent RDF data in low-level JavaScript libraries.
 
 ## Design elements and principles
-- We define data interfaces to represent **triples**, **quads**, **IRIs**, **blank nodes**, **literals** and **variables**.
+- We define data interfaces to represent **triples**, **quads**, **named nodes**, **blank nodes**, **literals** and **variables**.
 - Instances of the interfaces created with different libraries should be interoperable.
 - Interfaces do _not_ specify how instances are stored in memory.
-- Interfaces mandate specific pre-defined methods such as `.equals()`.
+- Interfaces mandate specific pre-defined methods such as `.sameTerm()`.
```

Comment:
Discussed in the 2015-11-27 meeting: we prefer to keep `.equals`, which checks both the type and the value.